external/anomaly/anomaly_segmentation/configs/configuration.py

```diff
@@ -95,5 +96,23 @@ class POTParameters(ParameterGroup):
             max_value=maxsize,
         )
 
+    @attrs
+    class NNCFOptimization(ParameterGroup):
```

Comment:
I don't think this is needed here. This class inherits from `BaseAnomalyConfig`, which already has the `NNCFOptimization` parameter group defined.

Reply:
But this class inherits from 'ConfigurableParameters'
https://github.com/openvinotoolkit/training_extensions/blob/develop/external/anomaly/anomaly_segmentation/configs/configuration.py#L35

Maybe this file should be removed, cause `BaseAnomalyClassificationConfig` does not used?

Reply:
Yes, you're right. That file was a leftover from duplicating the contents of the `anomaly_classification` folder when we first created the anomaly segmentation task. Thanks for spotting this!